tutorials/xr/openxr_spatial_entities.rst

```diff
@@ -0,0 +1,1710 @@
+                    new_scene.free()
+        else:
+            # Type of spatial entity tracker we're not supporting?
+            push_warning("OpenXR Spatial Entities: Unsupported anchor tracker " + tracker.get_name() + " of type " + tracker.get_class())
+
+        if not new_node:
+            # No scene defined or able to be instantiated? We're done!
+            return
+
+        # Setup and add to our scene.
+        new_node.tracker = tracker.name
+        new_node.pose = "default"
+        _managed_nodes[tracker] = new_node
+        add_child(new_node)
+
+        added_spatial_entity.emit(new_node)
+
+
+    # A new tracker was added to our XRServer.
+    func _on_tracker_added(tracker_name: StringName, type: int):
+        if type == XRServer.TRACKER_ANCHOR:
+            var tracker : XRTracker = XRServer.get_tracker(tracker_name)
+            if tracker and tracker is OpenXRSpatialEntityTracker:
+                _add_tracker(tracker)
+
+
+    # A tracked managed by XRServer was changed.
+    func _on_tracker_updated(_tracker_name: StringName, _type: int):
+        # For now we ignore this, there aren't changes here we need to react
+        # to and the instanced scene can react to this itself if needed.
+        pass
+
+
+    # A tracker was removed from our XRServer.
+    func _on_tracker_removed(tracker_name: StringName, type: int):
+        if type == XRServer.TRACKER_ANCHOR:
+            var tracker : XRTracker = XRServer.get_tracker(tracker_name)
+            if _managed_nodes.has(tracker):
+                # We emit this right before we remove it!
+                removed_spatial_entity.emit(_managed_nodes[tracker])
+
+                # Remove the node.
+                remove_child(_managed_nodes[tracker])
+
+                # Queue free the node.
+                _managed_nodes[tracker].queue_free()
+
+                # And remove from our managed nodes.
+                _managed_nodes.erase(tracker)
+
+Spatial anchors
+---------------
+
+Spatial anchors allow us to map real world location in our virtual world in such a way that the
+XR runtime will keep track of these locations and adjust them as needed.
+If supported anchors can be made persistent which means the anchors will be recreated in the correct
+location when your application starts again.
+
+You can think of use cases such as:
+- placing virtual windows around your space that are recreated when your application restarts
+- placing virtual objects on your table or on your walls and have them recreated
+
+Spatial anchors are tracked using :ref:`OpenXRAnchorTracker<class_OpenXRAnchorTracker>` objects
+registered with the XRServer.
+
+When needed the location of the spatial anchor will be updated automatically, the pose on the
+related tracker will be updated and thus the :ref:`XRAnchor3D<class_XRAnchor3D>` node will 
+reposition.
+
+When a spatial anchor has been made persistent a Universally Unique Identifier (or UUID) is
```

Comment:
This gets nitpicky since it's been a commonality in this PR to not include commas after dependent clauses but this is my second(ish) time noticing it. 

Original comment was

> Comma after persistent as "When a spatial anchor has been made persistent" is a dependent clause.

Reply:
Like this? `When a spatial anchor has been made persistent, a Universally Unique Identifier (or UUID) is`

Reply:
Yep, that would work.